src/content/docs/en/recipes/external-links.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Add icons to external links
+description: Learn how to add icons to external links in your Markdown files
+i18nReady: false
+type: recipe
+---
+
+Using a Rehype plugin, you can add icons to external links in your Markdown files, as well as open those links in a new tab.
+
+## Prerequisites
+- A project using Markdown for content pages.
+
+## Recipe
+
+1. Install the `rehype-external-links` plugin.
+
+    ```shell
+    npm install rehype-external-links
+    ```
+
+2. Add the plugin to your `astro.config.mjs` file. Pass the plugin options by passing an array with the plugin as the first item and the options as the second item. Set the content type to `text`, or `raw` to use HTML as the content.
+
+    ```ts
+    // ...
+    import rehypeExternalLinks from 'rehype-external-links';
+
+    export default defineConfig({
+      // ...
+      markdown: {
+        rehypePlugins: [
+          [
+            rehypeExternalLinks,
+            {
+              target: '_blank',
+              rel: ['noopener', 'noreferrer'],
+              content: { type: 'text', value: ' 🔗' }
```

Comment:
```suggestion
              content: { type: 'text', value: ' 🔗' }
```

For the purposes of this example, we don't need to set these attributes to add an icon.

Additionally, the package warns us to not set `target`, and setting `rel` to these values is only appropriate if `target` is `_blank`.